tools/clang/test/HLSLFileCheck/shader_targets/raytracing/subobjects_raytracingPipelineConfig1.hlsl

```diff
@@ -1,4 +1,9 @@
-// RUN: %dxilver 1.5 | %dxc -T lib_6_3 %s | FileCheck %s
+// RUN: %dxilver 1.5 | %dxc -T lib_6_3 %s | FileCheck %s -check-prefixes=CHECK
```

Comment:
Why check-prefixes here?

Reply:
Oops, leftover from intermediate change, I can remove it.